### PR TITLE
fix(vicon.ts): svg size didn't match default icon size

### LINKED ISSUE #10480: [Bug Report] [VTextField] Clear icon (svg)  mis-alignment
### Environment
**Vuetify Version:** 2.2.10
**Last working version:** 2.2.8
**Vue Version:** 2.6.11
**Browsers:** Chrome 79.0.3945.130
**OS:** Mac OS 10.15.3

### Steps to reproduce
1. Set 'mdiSvg' as the fonts option for Vuetify
1. Set a v-text-field 'clearable' 

### Expected Behavior
icons in a v-text-field should stay aligned. The Clear icon, in this case, should move up a little.

### Actual Behavior
The SVG icons appear lower than their FONT counterparts.

### Reproduction Link
<a href="https://codepen.io/myleslee/pen/OJVJRwB?editors=1010" target="_blank">https://codepen.io/myleslee/pen/OJVJRwB?editors=1010</a>

### PR DESCRIPTION
**Pull request by rachael-smith**
default font size was 24 but svg size was set to 32 making alignment for svgs off if not specified

## Description
default font size was 24 but svg size was set to 32 making alignment for svgs off if not specified
resolves bug report #10480

## Motivation and Context
fixes #10480

## How Has This Been Tested?
went through snapshot test and tested locally by testing default and changing size in v-icon tag

## Markup:
can throw this in playground https://codepen.io/myleslee/pen/OJVJRwB?editors=1010 (also pasted below)

<details>

```vue
<template>
  <v-container>
<div id="app">
  <v-app id="inspire">
    <v-form>
      <v-container>
        <v-row>

          <v-col cols="12" sm="6">
            <v-text-field v-model="message1" label="Regular" clearable append-icon="event">
              <template v-slot:prepend>
                <v-icon>event</v-icon>
              </template>
              <template v-slot:prepend-inner>
                <v-icon @click="alert('a')">
                  M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z
                </v-icon>
              </template>
              <template v-slot:append-outer>
                <v-icon @click="alert('a')">
                  clear
                </v-icon>
              </template>
            </v-text-field>
          </v-col>
        </v-row>
      </v-container>
    </v-form>
  </v-app>
</div>
  </v-container>
</template>

<script>
export default {
  data () {
    return {
      message1: 'icons in lower position are svgs',
      message2: 'Hey!',
      message3: 'Hey!',
      message4: 'Hey!',
    }
  }, methods: {
    alert(val) {
      alert(val)
    }
  }
}
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by rachael-smith** [comment]
same as https://github.com/vuetifyjs/vuetify/pull/10497 just cleaned up

**Comment by rachael-smith** [comment]
updated vchip which was also having the same issue as pointed out by @funkybg

File: packages/vuetify/src/components/VChip/_variables.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 @import '../../styles/tools/_functions.sass';
 
 $chip-avatar-size: 24px !default;
-$chip-close-size: 18px !default;
+$chip-close-size: 24px !default;
```
Comment (KaelWD) [inline packages/vuetify/src/components/VChip/_variables.scss]: https://material.io/components/chips/#specs

Comment (rachael-smith) [reply packages/vuetify/src/components/VChip/_variables.scss]: everything else is defaulted to 24px which is why there are alignment issues with chip-close... should everything be defaulted to 18? or just the chips?

Comment (KaelWD) [reply packages/vuetify/src/components/VChip/_variables.scss]: Just chips are 18

Comment (rachael-smith) [reply packages/vuetify/src/components/VChip/_variables.scss]: it okay if i back out the chips and push through the first issue and handle the chips in a separate commit? if not i can do it in this PR just going to be a day or so

Comment (KaelWD) [reply packages/vuetify/src/components/VChip/_variables.scss]: a day or so is fine

Comment (rachael-smith) [reply packages/vuetify/src/components/VChip/_variables.scss]: updated

**Comment by rachael-smith** [comment]
@KaelWD should be good now

File: packages/vuetify/src/components/VChip/VChip.ts [hunk 2]
```diff
@@ -145,6 +145,9 @@ export default mixins(
         props: {
           right: true,
         },
+        attrs: {
```
Comment (KaelWD) [inline packages/vuetify/src/components/VChip/VChip.ts]: did you try `props: { size: 18 }`?

Comment (KaelWD) [reply packages/vuetify/src/components/VChip/VChip.ts]: Or use css to override the width/height/font-size on the `.v-chip__close` class if we want to keep the sass variable.

Comment (rachael-smith) [reply packages/vuetify/src/components/VChip/VChip.ts]: @KaelWD I had tried fontSize and it didn't work but size did should be updated now :)

File: packages/vuetify/src/components/VIcon/VIcon.ts [hunk 3]
```diff
@@ -150,20 +150,19 @@ const VIcon = mixins(
           width: fontSize,
         } : undefined,
       }
+      console.log(fontSize)
```
Comment (DRoet) [inline packages/vuetify/src/components/VIcon/VIcon.ts]: remove logging

File: packages/vuetify/src/components/VChip/VChip.ts [hunk 4]
```diff
@@ -170,7 +171,6 @@ export default mixins(
   render (h): VNode {
     const children = [this.genContent()]
     let { tag, data } = this.generateRouteLink()
-
```
Comment (DRoet) [inline packages/vuetify/src/components/VChip/VChip.ts]: revert all these spacing changes

**Comment by rachael-smith** [comment]
@KaelWD should be good

**Pull request merged**